mkdocs.yml

```diff
@@ -12,7 +12,9 @@ nav:
   - Home: index.md
   - Documentation:
     - Overview: docs/overview.md
-    - Installing: docs/installing.md
```

Comment:
Would you mind testing out: https://pypi.org/project/mkdocs-redirects/ so that users don't get a 404 if they go to the old page and instead they're redirected to the new page?

Can we also rename these sections to:
```
- Getting started:
  - Install Copilot
  - (Optional) Verify the installation
  - Deploy your first application
```
(basically merging Configuration and Getting Started)

Reply:
done!

Reply:
nit: can the files be moved to `docs/getting-started/install.md`, `docs/getting-started/verify.md`, `docs/getting-started/first-app-tutorial.md`

Also let me know if redirects work 🙏